pareto/utilities/get_data.py

```diff
@@ -728,6 +733,13 @@ def get_display_units(input_sheet_name_list, user_units):
         "MinTreatmentFlow": user_units["volume"],
         "MinResidualQuality": "",
         "DesalinationSurrogate": "kg/s",
+        "AirEmissionCoefficients": user_units["mass"] + "/X",
```

Comment:
I am not sure if I understand this correctly. Can you please explain this.

Reply:
The parameters which are defined on the "AirEmissionCoefficients" tab always have mass in the numerator, but the denominator can vary. So I used "/X" as a generic stand-in here.

Reply:
Oh ok. Got it thanks.